packages/core/utils/src/dml/types.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * The supported data types
+ */
+export type KnownDataTypes =
+  | "string"
+  | "boolean"
+  | "enum"
+  | "number"
+  | "date"
+  | "dateTime"
+  | "json"
+  | "any"
+
+/**
+ * The meta-data returned by the schema parse method
+ */
+export type SchemaMetaData = {
```

Comment:
```suggestion
export type SchemaMetadata = {
```

Reply:
Okay, I have always wrote it as `metaData`. So it should be `metadata` and `SchemaMetadata` right?

Reply:
Yeah, that's how it is everywhere (and TBH how it's commonly typed, as it is a single word in the dictionary 😄 )

Reply:
Good to know 😄